### PR TITLE
Ansible Questions Added 

### PR DESCRIPTION
**Pull request by Rahulsharma0810**
Ansible Questions

1 Ansible  Answer updated
1 Ansible Question Added

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4490,6 +4598,10 @@ kubectl run nginx --image=nginx --restart=Never --port 80 --expose
 <summary>Describe in detail what the following command does <code>kubectl create deployment kubernetes-httpd --image=httpd</code></summary><br><b>
 </b></details>
 
+<details>
```
Comment (bregman-arie) [inline README.md:4601]: You already added this

File: README.md [hunk 2]
```diff
@@ -4127,6 +4179,62 @@ Docker daemon creates a new container by using the image it downloaded
 Docker daemon redirects output from container to Docker CLI which redirects it to the standard output
 </b></details>
 
+<details>
```
Comment (bregman-arie) [inline README.md:4182]: You already added this

**Review by bregman-arie — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Rahulsharma0810) [reply README.md:4601]: I didn't get you, the 4601 `<details>` should not be there? 

below is the sytax
```
<details>
<summary>Why to create kind deployment, if pods can be launched with replicaset ? </summary><br><b>
</b></details>
```

Comment (bregman-arie) [reply README.md:4601]: I'm just saying that these questions (Kubernetes) are already added by you in a previous PR. I think you need to rebase your PR to only add the Ansible questions.

**Pull request closed without merging**